src/main/kotlin/com/github/devcordde/devcordbot/Launcher.kt

```diff
@@ -76,6 +78,11 @@ suspend fun main(args: Array<String>) {
     val kord = Kord(config.discord.token) {
         httpClient = HttpClient(CIO)
         intents = Intents.nonPrivileged + Intent.GuildMembers
+        cache {
+            guilds { cache, description ->
+                MapEntryCache(cache, description, MapLikeCollection.concurrentHashMap())
+            }
+        }
```

Comment:
This still won't have any effect since guild caching is a thing that is enabled by default